demo/node/node_subprocess.go

```diff
@@ -396,10 +346,8 @@ func (n *NodeProc) GetGroup() *key.Group {
 }
 
 func (n *NodeProc) ChainInfo(_ string) bool {
-	args := []string{"get", "chain-info"}
-	if n.tls {
-		args = append(args, pair("--tls-cert", n.certPath)...)
-	}
+	args := []string{"show", "chain-info", "--control", n.ctrl}
```

Comment:
hmm I guess the `node_subprocess` integration test should probably live in the CLI repo now, though that seems like a pain

Reply:
Yeah :/ I had to use workarounds in many places, including calling `curl` at some point in the GetBeacon func instead of using the CLI.